### DISCUSSION TITLE
Closing a session exits tmux

### DISCUSSION THREAD

**Discussion by mok0** [joshmedeski/sesh#120] [General]
I might be doing something wrong, or expecting something not possible, but when in the `sesh` dialog box (prefix T) and I select a session to kill with C-d, tmux just exits. I would expect only that one session to close, but the others remain, and that I remain inside tmux. I can of course restart tmux, and the other sessions are still there, but it's just confusing.

**Comment by mok0** [comment]
I solved my problem, sorry for the noise :-) I found I had to add to my `tmux.conf` this option

    set -g detach-on-destroy off 

I suddenly remembered Josh showed that in one of the videocasts.

**Reply by joshmedeski** [reply]
I also have recommended tmux settings recommended on the README:
https://github.com/joshmedeski/sesh?tab=readme-ov-file#recommended-tmux-settings
